content/en/security_monitoring/detection_rules/_index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Detection Rules
+kind: documentation
+further_reading:
+- link: "security_monitoring/default_rules"
+  tag: "Documentation"
+  text: "Explore default detection rules"
+- link: "/security_monitoring/detection_rules/security_monitoring"
+  tag: "Documentation"
+  text: "Follow the creating new Security Monitoring rules guide"
+- link: "/security_monitoring/detection_rules/compliance_monitoring"
+  tag: "Documentation"
+  text: "Follow the creating new Compliance Monitoring rules guide"
+---
+
+## Overview
+
+Detection Rules define conditional logic that is applied to all ingested logs and cloud configurations. When at least one case defined in a rule is matched over a given period of time, Datadog generates a Security Signal.
+
+{{< img src="security_monitoring/detection_rules/detection_rules.png" alt="Detection Rules" width="75%">}}
+
+Security Monitoring uses Log Detection to analyze ingested logs in real time. Compliance Monitoring uses Cloud Configuration to scan the state of your cloud environment. For each monitoring option, there are [default detection rules][1] that work out-of-the-box with integration configuration. You can also create new rules to tailor to your environment.
```

Comment:
```suggestion
Security Monitoring uses Log Detection to analyze ingested logs in real-time. Compliance Monitoring uses Cloud Configuration to scan the state of your cloud environment. For each monitoring option, there are [default detection rules][1] that work out-of-the-box with integration configuration. You can also create new rules to tailor to your environment.
```
I am not sure what you mean by "Cloud Configuration" here. Is this some sort of spec we can link to? Or can we be more specific?